atomics/T1059.004/T1059.004.yaml

```diff
@@ -254,3 +254,29 @@ atomic_tests:
   executor:
     command: awk 'BEGIN {system("/bin/sh &")}'
     name: sh
+- name: Creating shell using cpan command
+  auto_generated_guid: 
+  description: |-
+    cpan lets you execute perl commands with the ! command. It can be used to break out from restricted environments by spawning an interactive system shell.
+    Reference - https://gtfobins.github.io/gtfobins/cpan/
+  supported_platforms:
+  - linux
+  - macos
+  executor:
+    command: |-
+      cpan
+      ! exec '/bin/bash &'
```

Comment:
Hello @nish221b-bs . Thank you for your contribution. This test doesnt work when it is non interactive (used with executors like Invoke-AtomicRedTeam). Can you change this test to 

`echo '! exec "/bin/sh &"' | PERL_MM_USE_DEFAULT=1  cpan`

so that it works with executors ? 

Note: Added `PERL_MM_USE_DEFAULT=1` to accept default values to configure cpan without waiting for user inputs.

Reply:
@cyberbuff yeah sure will make the change. Thank you for the update.